lib/omnibus/packagers/base.rb

```diff
@@ -127,6 +127,25 @@ def install_dir
     #
     expose :windows_safe_path
 
+    #
+    # Skip this packager during build process
+    #
+    # @example
+    #   skip_package true
+    #
+    # @param [TrueClass, FalseClass] value
+    #   whether to delay validation or not
+    #
+    # @return [TrueClass, FalseClass]
+    #   whether to skip this packager type or not
+    def skip_packager(val = false)
```

Comment:
I would propose we change the signature of this method to be `def skip_packager()`. IMO if the user is adding `skip_packager` to their packaging definition that is enough indication of what they want to do. The `true` or `false` does not seem necessary

Reply:
I like the idea, however there are other options exposed on `msi` packagers that I based this on. 

```
git grep "def.*\(val = false\)"
lib/omnibus/packagers/msi.rb:    def bundle_msi(val = false)
lib/omnibus/packagers/msi.rb:    def fast_msi(val = false)
```

I believe consistency is key to adoption of function.  I offered another similar option in #842 and `master` currently offers the above two instances.

If a refactor is desired can I consolidate my 2 PRs and expand to refactor the other two instances which would make this a breaking api change, *or* this and #843 can land as is, then offer a refactor PR that includes the required version bump.